site/class-1-bar/youmghar/readme.md

```diff
@@ -0,0 +1,40 @@
+# Barchart assignment
+
+### This is a basic barchart with a hover which changes the colors of the bars.
+
+##Background
+
+This barchart works with D3. And loads external data into its HTML. Which is styled with a seperate CSS file. All the visuals are build up out off SVG.
+
+##Data
+
+The data that is visualized in the barchart is just test data. The data consists out of A letter which is placed on the x axis and a frequency which is shown on the y axis.
+
+##Features
+
+D3 API = https://d3js.org/d3.v4.min.js
```

Comment:
This is okay for now, but in the future you should describe all the separate features you’re using, not just linking to the whole library! (also: you can create [links in markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)).